.ci/install_qemu_experimental.sh

```diff
@@ -26,7 +27,11 @@ qemu_experimental_latest_build_url="${jenkins_url}/job/qemu-experimental-nightly
 bindir="${DESTDIR:-}/usr/bin"
 libexecdir="${DESTDIR:-}/usr/libexec/"
 
-QEMU_PATH="/opt/kata/bin/qemu-system-x86_64-experimental"
+if [ "${KATA_BUILD_QEMU_TYPE}" == "snp" ]; then
+	QEMU_PATH="/opt/confidential-containers/bin/qemu-system-x86_64-snp"
```

Comment:
In a VM where I ran this script I got the qemu for snp installed with another name:

```
$ crudini --get /opt/confidential-containers/share/defaults/kata-containers/configuration-qemu-snp.toml hypervisor.qemu path
"/opt/confidential-containers/bin/qemu-system-x86_64-snp-experimental"
$ ls /opt/confidential-containers/bin/
containerd-shim-kata-v2  kata-collect-data.sh  kata-monitor  kata-runtime  qemu-system-x86_64-snp-experimental
```

Maybe some code on this install_qemu_experimental.sh script is bogus?

Reply:
Good catch @wainersm.
I have removed the bogus code in build_qemu_experimental.sh